### ISSUE TITLE
You broke everything in 6.3.1

### ISSUE THREAD

**Issue by juleek** [django-ckeditor/django-ckeditor#708]
This the change:

```diff
git diff 6.3.0..6.3.1
diff --git a/ckeditor/__init__.py b/ckeditor/__init__.py
index 5ab5c88..7b8ac80 100644
--- a/ckeditor/__init__.py
+++ b/ckeditor/__init__.py
@@ -1,2 +1,2 @@
-VERSION = (6, 3, 0)
+VERSION = (6, 3, 1)
 __version__ = ".".join(map(str, VERSION))
diff --git a/ckeditor/fields.py b/ckeditor/fields.py
index d0a7f53..c8c55b5 100644
--- a/ckeditor/fields.py
+++ b/ckeditor/fields.py
@@ -35,13 +35,15 @@ class RichTextFormField(forms.fields.CharField):
         *args,
         **kwargs
     ):
-        kwargs.update(
-            {
+        super().__init__(
+            *args,
+            **{
                 "widget": CKEditorWidget(
                     config_name=config_name,
                     extra_plugins=extra_plugins,
                     external_plugin_resources=external_plugin_resources,
-                )
+                ),
+                # Let developers override the kwargs inserted above
+                **kwargs,
             }
         )
-        super().__init__(*args, **kwargs)

```

Before the change you had this code:

```python
        kwargs.update(
            {
                "widget": CKEditorWidget(
                    config_name=config_name,
                    extra_plugins=extra_plugins,
                    external_plugin_resources=external_plugin_resources,
                )
            }
        )
        super().__init__(*args, **kwargs)
```

## while the new code:

```python
            **{
                "widget": CKEditorWidget(
                    config_name=config_name,
                    extra_plugins=extra_plugins,
                    external_plugin_resources=external_plugin_resources,
                ),
                # Let developers override the kwargs inserted above
                **kwargs,
            }
```
allows the caller (which specifies `TextArea` as a widget) to override your widget (`CKEditorWidget`).

P.S.
The caller is here:

```python
    def formfield(self, **kwargs):
        # Passing max_length to forms.CharField means that the value's length
        # will be validated twice. This is considered acceptable since we want
        # the value in the form field (to pass into widget for example).
        return super().formfield(
            **{
                "max_length": self.max_length,
                **({} if self.choices is not None else {"widget": forms.Textarea}),
                **kwargs,
            }
        )
```

**Comment by matthiask** [comment]
Thanks. That was truly a bad fix.

**Comment by ddabble** [comment]
Oh whoops, sorry about that! I had apparently not investigated thoroughly enough before making that change 😅

I made a (hopefully) improved fix in #712; @juleek do things still work as expected, with this fix? 🙂
